saleor/graphql/order/types.py

```diff
@@ -1031,6 +1046,17 @@ def wrap_shipping_method_with_channel_context(data):
 
     @staticmethod
     @traced_resolver
+    def resolve_delivery_method(root: models.Order, info):
+        if root.shipping_method_id:
+            return Order.resolve_shipping_method(root, info)
```

Comment:
I believe, you can add `@classmethod` instead of `staticmethod` and call `cls` here.

Reply:
Yep, done.